src/main/java/org/pac4j/vertx/handler/impl/Pac4jAuthHandlerOptions.java

```diff
@@ -26,6 +26,7 @@
     private String clientName = "";
```

Comment:
`clientName`, `authorizerName` and `matcherName` have been replaced by `clients`,  `authorizers` and `matchers` for quite some time (because we may have a list of values separated by commas in the provided string). I'm not sure we still need this `Pac4jAuthHandlerOptions` component.